packages/sonarwhal/src/lib/cli/analyze.ts

```diff
@@ -26,33 +28,46 @@ const debug: debug.IDebugger = d(__filename);
  * ------------------------------------------------------------------------------
  */
 
-const confirmLaunchInit = (): inquirer.Answers => {
-    debug(`Initiating launch init confirm.`);
+const askForConfirm = (question: string): inquirer.Answers => {
+    debug(`Asking for confirmation`);
 
-    const question: Array<object> = [{
-        message: `A valid configuration file can't be found. Do you want to create a new one?`,
+    const questions: inquirer.Questions = [{
+        message: question,
         name: 'confirm',
         type: 'confirm'
     }];
 
-    return inquirer.prompt(question);
+    return inquirer.prompt(questions);
 };
 
 const askUserToCreateConfig = async (): Promise<boolean> => {
-    const launchInit: inquirer.Answers = await confirmLaunchInit();
+    const question: string = `Do you want to create a new configuration?`;
+    const launchInit: inquirer.Answers = await askForConfirm(question);
 
     if (!launchInit.confirm) {
         return false;
     }
 
     const { default: initSonarwhalrc } = await import('./wizards/init');
 
-    await initSonarwhalrc();
+    const sonarwhalrcCreated = await initSonarwhalrc();
+
+    if (!sonarwhalrcCreated) {
+        return false;
+    }
+
     logger.log(`Configuration file .sonarwhalrc was created.`);
 
     return true;
 };
 
+const askUserToUseDefaultConfiguration = async (): Promise<boolean> => {
+    const question: string = `A valid configuration file can't be found. Do you want to use the default configuration?`;
```

Comment:
We should add a link about the default configuration. Can you update the `user-guide` document and add a section "default configuration" saying that we use the `web-recommended` one, etc.? Message should be:

> [...] Do you want to use the default configuration? To know more about the default configuration visit: https://blablabla